### PR TITLE
Use SQS parameter message group id for sqs targets in events rule.

### PR DESCRIPTION
**Pull request by irahulranjan**
- addresses https://github.com/localstack/localstack/issues/3402

### CODE REVIEW

File: localstack/utils/aws/aws_stack.py [hunk 1]
```diff
@@ -536,7 +536,15 @@ def send_event_to_target(arn, event):
     elif ':sqs:' in arn:
         sqs_client = connect_to_service('sqs')
         queue_url = get_sqs_queue_url(arn)
-        sqs_client.send_message(QueueUrl=queue_url, MessageBody=json.dumps(event))
+
+        msg_group_id = None
```
Comment (whummer) [inline localstack/utils/aws/aws_stack.py:540]: nitpick: I think we can slightly simplify this (and reduce duplication a bit):
```
        msg_group_id = (target_attributes or {}).get('MessageGroupId')
        kwargs = {'MessageGroupId': msg_group_id} if msg_group_id else {}
        sqs_client.send_message(QueueUrl=queue_url, MessageBody=json.dumps(event), **kwargs)
```

File: localstack/utils/aws/aws_stack.py [hunk 2]
```diff
@@ -546,6 +554,12 @@ def send_event_to_target(arn, event):
         LOG.info('Unsupported Events rule target ARN "%s"' % arn)
 
 
+def get_events_target_attributes(target):
+    # added for sqs, if needed can be moved to an if else
+    # block for multiple targets
+    return target.get('SqsParameters')
```
Comment (whummer) [inline localstack/utils/aws/aws_stack.py:560]: Which other parameter types could occur here? Wondering if we should add the remaining ones as part of this PR as well..

Comment (irahulranjan) [reply localstack/utils/aws/aws_stack.py:560]: KinesisParameters
RunCommandParameters
EcsParameters
BatchParameters
SqsParameters
HttpParameters
HeaderParameters
QueryStringParameters
RedshiftDataParameters

**Comment by whummer** [comment]
Looks great, thanks @irahulranjan 👍

**Pull request merged**